helm/.helm-template.yaml

```diff
@@ -0,0 +1,841 @@
+# ---------------------------------------------------------------------
```

Comment:
Can you name it `.values-template.yaml` or something to indicate it's used to generate the values.yaml?

Reply:
Done